### PR TITLE
Db/edit mode design

### PR DESCRIPTION
**Pull request by pawellysy**

### CODE REVIEW

File: samples/dashboard/demos/climate/demo.css [hunk 1]
```diff
@@ -116,3 +116,57 @@
...
+.hd-edit-toolbar > .hd-edit-toolbar-cell-outline {
+    border: 2px solid #241d3a;
+}
+
+.hd-edit-toolbar > .hd-edit-toolbar-row-outline {
+    border: 2px solid #646479;
+}
+
+.hd-edit-menu.hd-edit-toolbar-row {
+    background: #646479;
```
Comment (sebastianbochan) [inline samples/dashboard/demos/climate/demo.css:171]: I see some colors (like this `#646479` ) in a few places, I suggest extracting them to separate variables and using then.

**Review by sebastianbochan — CHANGES_REQUESTED** [review]

**Review by karolkolodziej — CHANGES_REQUESTED** [review]
Please adjust styling when dragging elements.

**Review by karolkolodziej — APPROVED** [review]

**Review by bre1470 — APPROVED** [review]
FYI: After drop and leaving edit mode some container marking does not vanish. Not sure if this falls into the scope of this PR.

Screenshot 2023-01-23 at 14 15 11

**Comment by pawellysy** [comment]
@bre1470  I will report as a seperate bug 👍

**Pull request merged**